lib/ansible/plugins/connection/winrm.py

```diff
@@ -76,9 +73,11 @@
         description:
            - List of winrm transports to attempt to use (ssl, plaintext, kerberos, etc)
            - If None (the default) the plugin will try to automatically guess the correct list
-           - The choices available depend on your version of pywinrm
+           - This can be set to C(basic), C(certificate), C(ntlm), C(kerberos), C(credssp).
+           - It can also be set to C(plaintext) or C(ssl) but these values will use use C(basic)
+             auth but over a HTTP or HTTPS connection respectively.
         type: list
-        elements: string
+        elements: str
```

Comment:
Is there a reason not to use `choices` to indicate the valid options covered in the description?